ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionProcessor.java

```diff
@@ -288,16 +288,17 @@ public TransactionProcessingResult processTransaction(
 
       final MutableAccount senderMutableAccount = sender.getMutable();
       final long previousNonce = senderMutableAccount.incrementNonce();
-      final Wei transactionGasPrice =
-          feeMarket.getTransactionPriceCalculator().price(transaction, blockHeader.getBaseFee());
-      final Wei dataGasPrice =
-          feeMarket.getTransactionPriceCalculator().dataPrice(transaction, blockHeader);
       LOG.trace(
           "Incremented sender {} nonce ({} -> {})",
           senderAddress,
           previousNonce,
           sender.getNonce());
 
+      final Wei transactionGasPrice =
```

Comment:
Did moving this to the other side of the trace really have an impact?  Can it be rolled back for clarity?

Reply:
My intent was to move the log of the nonce just after the statement that changed it, no behavior change of course

Reply:
My intent was to move the log of the nonce just after the statement that changed it, no behavior change of course